include/engine/api/json_factory.hpp

```diff
@@ -42,9 +42,9 @@ std::string modeToString(const extractor::TravelMode mode);
 
 } // namespace detail
 
-template <typename ForwardIter> util::json::String makePolyline(ForwardIter begin, ForwardIter end)
+template <typename ForwardIter> util::json::String makePolyline(ForwardIter begin, ForwardIter end, double polylinePrecision = 1e5)
```

Comment:
The polylinePrecision is a factor we do know at compile time (either 5 or 6 depending on the callsite). This function is rather performance critical, specially for long geometries. This leads me to suggest that we make it a template parameter like:

`template <typename ForwardIter, unsigned PRECISION=1e5>  makePolyline`.